### DISCUSSION TITLE
Is there an OpenAPI generator to produce interfaces?

### DISCUSSION THREAD

**Discussion by vladaman** [quarkusio/quarkus#27572] [Q&A] [answered]
We have API First driven development and we'd like to generate interfaces base on openapi.yaml spec file. Is there any generator which would generate interfaces compatible with Microprofile + quarkus-resteasy-jackson ?

[a comment by an automated account is not shown]

**Comment by stephan-strate** [comment, marked as answer]
Theres a quarkiverse extension. But there are some known [limitations](https://github.com/quarkiverse/quarkus-openapi-generator#known-limitations).
https://github.com/quarkiverse/quarkus-openapi-generator

**Reply by phillip-kruger** [reply]
Yes, above is all I know about.
